nick_testing.py

```diff
@@ -0,0 +1,36 @@
+from gladier.client import GladierClient
+# This is a HACK to enable glaider logging
+import gladier.tests
+from pprint import pprint
+
+
+class SSXReprocessing(GladierClient):
+    client_id = 'e6c75d97-532a-4c88-b031-8584a319fa3e'
+    gladier_tools = [
+        'tools.publish.SSXPilot',
+    ]
+
+    flow_definition = 'tools.publish.SSXPilot'
```

Comment:
This is interesting. I will try to fix mine to look the same later.

Reply:
Ahh, this also requires treating the gladier_kanzus project as a 'package', where 'tools' is the package name. Maybe we should have a different top level folder structured like 'gladier_kanzus/tools/`?